greenplumpython/expr.py

```diff
@@ -64,6 +76,23 @@ def __str__(self) -> str:
         return str(self.left) + " " + self.operator + " " + str(self.right)
 
 
+class UnaryExpr(Expr):
+    def __init__(self, operator: str, right: Expr, as_name: Optional[str] = None):
+        super().__init__(as_name=as_name)
+        self.operator = operator
+        self.right = right
+
+    def __str__(self) -> str:
+        if self.operator == "NOT":
+            return "NOT(" + str(self.right) + ') AS "Not(' + str(self.right) + ')"'
+        if self.operator == "ABS":
+            return "ABS(" + str(self.right) + ') AS "Abs(' + str(self.right) + ')"'
+
```

Comment:
we need to raise an exception for the unsupported operators. And, don't forget to add a test to check the exception.